datastore/tests/unit/test_transaction.py

```diff
@@ -19,6 +19,9 @@
 
 class TestTransaction(unittest.TestCase):
 
+    def setUp(self):
+        self.project = 'PROJECT'
```

Comment:
Please just use a class constant instead of `setUp`. (In general we avoid `setUp` and `tearDown` at all costs because we'd prefer state to be "obvious" like a class constant or local to each test case.)